### DISCUSSION TITLE
Invest more in UI

### DISCUSSION THREAD

**Discussion by navanshu** [mnfst/llm-gateway#99] [Feedback]
As a developer it looks okay, but a better UI and feel can open more opportunities for you, rest is good.

**Comment by SebConejo** [comment]
Hello @navanshu, Thank you for your feedback. 

Could you clarify which aspect you're referring to – the admin panel, CLI experience, documentation, or something else? What specific improvements are you looking for ?

**Reply by navanshu** [reply]
@SebConejo Improvements on homepage and the admin panel is necessary, you can skip documentation for now.

For the homepage the strong section is only the editor and video section

The admin panels color needs improvement.

These are some basic pointers.

**Comment by boustanihani** [comment]
Maybe take a look at pocketbase and borrow some ideas => https://pocketbase.io/demo/

**Reply by SebConejo** [reply]
@boustanihani Manifest takes a different approach. The YAML file is for developers, and it generates an admin panel for content editors. It is not a no code tool used by all the users. That changes everything. Different needs, different developer experience and you do not need the same type or level of API documentation at the same stage for example.

In any case, we are working on it those days

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
